test/e2e/pkg/manifest.go

```diff
@@ -46,6 +46,10 @@ type Manifest struct {
 
 	// Nodes specifies the network nodes. At least one node must be given.
 	Nodes map[string]*ManifestNode `toml:"node"`
+
+	// KeyType sets the curve that will be used by validators.
+	// Options are ed25519 & secp256k1
+	KeyType string `toml:"key_type"`
```

Comment:
For now this is used to set all keys of a validator. I did some testing, using two different curves caused some issues. Need to look deeper into it.